src/Platform.c

```diff
@@ -464,24 +464,162 @@ void PltWaitForConditionVariable(PLT_COND* cond, PLT_MUTEX* mutex) {
 #endif
 }
 
-uint64_t PltGetMillis(void) {
+//// Begin SDL time emulation
+
+// These functions mostly emulate SDL_GetTicks64(), because this library doesn't
+// use SDL, but timing data needs to be compatible with SDL-supplied data in moonlight-qt.
+
+static bool has_monotonic_time = false;
+static bool ticks_started = false;
+
 #if defined(LC_WINDOWS)
-    return GetTickCount64();
-#elif defined(CLOCK_MONOTONIC) && !defined(NO_CLOCK_GETTIME)
-    struct timespec tv;
 
-    clock_gettime(CLOCK_MONOTONIC, &tv);
+static LARGE_INTEGER start_ticks;
+static LARGE_INTEGER ticks_per_second;
+
+void Plt_TicksInit(void) {
+    if (ticks_started) {
+        return;
+    }
+    ticks_started = true;
+    BOOL rc = QueryPerformanceFrequency(&ticks_per_second);
+    QueryPerformanceCounter(&start_ticks);
+}
+
+uint64_t Plt_GetTicks64_us(void) {
```

Comment:
I think I'd prefer to match the external API nomeclature here `PltGetMicroseconds()`, especially since I don't think we should promise the SDL timebase semantics and we don't use underscores in our API names elsewhere.

Reply:
Sounds good, I will put back `PltGetMillis()` and rename this to `PltGetMicroseconds()`.